scripts/template/Component.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { MarigoldProvider } from '@marigold/system';
+import { COMP } from '@marigold/components';
+
+const theme = {};
+
+test('supports default variant and themeSection', () => {});
+
+test('accepts other variant than default', () => {});
+
+test('renders correct HTML element', () => {});
+
+test('supports custom prop', () => {});
```

Comment:
Do we call it "other variant" or "custom prop" overall or we have to change here to the name of the variant/prop in de actual component? @sebald @viktoria-schwarz

Reply:
this test should test if e.g. in Icon it supports the fill or size prop, so you need to insert the prop name here if there is one :)

Reply:
Yes, I think also it's better to add the prop name 👍